src/connectors/truckersfm.js

```diff
@@ -11,3 +15,10 @@ Connector.trackSelector = '.player-title-text';
 Connector.playButtonSelector = '.pause-btn > .fa-play';
 
 Connector.onReady = Connector.onStateChanged;
+
+Connector.applyFilter(filter);
+
+function formatArtists(text) {
+	const artist = text.split(',');
```

Comment:
I forgot to mention this way won't work for artists who have a comma symbol in their names.

Reply:
Maybe I guess, honestly, I've never seen an artist with a comma but I guess in the name of redundancy.

Maybe we make it match with space as the layout is like Artist1, Artist2. So we just use .split(', ')

Not sure if this would account for the space though.

Reply:
> I've never seen an artist with a comma but I guess in the name of redundancy.

I have few examples of real names containing comma character (see https://github.com/web-scrobbler/web-scrobbler/issues/2608#issuecomment-731719288 for details).

> Maybe we make it match with the space as the layout is like Artist1, Artist2. So we just use .split(', ')

Using `, ` as a split symbol won't resolve this. In other connectors I use the following approach:
https://github.com/web-scrobbler/web-scrobbler/blob/d3a369b18d908deb375a763ca83e43ddb92a47c4/src/connectors/monstercat.js#L34-L42

Not sure if TruckersFM website provides artists as separate nodes, though.